docs/lib/content/configuring-npm/package-json.md

```diff
@@ -1191,6 +1209,24 @@ default.
 See [`config`](/using-npm/config) to see the list of config options that
 can be overridden.
 
+### type
+
+> See Node.js documentation on `type` [here](https://nodejs.org/api/packages.html#type).
+
+Property that governs if Node.js will interpret package as [CommonJS (cjs)](https://nodejs.org/docs/latest/api/modules.html#modules-commonjs-modules) or [ECMAScript Module (esm)](https://nodejs.org/docs/latest/api/esm.html).
+
+When this field is undefined it is interpreted as the value `commonjs`.
+
+### types
+
+See TypeScript documentation on `types` [here](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html).
+
+Used by npmjs.org to display `ts` badege for a given package.
+
+### typings
+
+Alias of [types](#types).
```

Comment:
i think this is actually a typescript-deprecated field from TS < v3 or v2.

Reply:
From the ts docs:

> Note that the "typings" field is synonymous with types, and could be used as well.

We do honor it.

If you can find deprecation docs I'm down to add `(deprecation)`. I still think it should be documented.

Reply:
https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html implies it's just an alias, so maybe it's not actually deprecated.